lms/djangoapps/certificates/queue.py

```diff
@@ -237,8 +243,15 @@ def add_cert(self, student, course_id, course=None, forced_grade=None, template_
             status.unverified,
         ]
 
-        cert_status = certificate_status_for_student(student, course_id)['status']
+        cert_status_dict = certificate_status_for_student(student, course_id)
+        cert_status = cert_status_dict['status']
```

Comment:
get would be more appropriate to avoid the exception if status is't in the dict.

Reply:
@adeelehsan It will always be in the dict.

Reply:
never mind I changed it to use get.